routes/urls.js

```diff
@@ -705,6 +723,45 @@ module.exports = async function (app) {
         };
       }
 
+      return { status: 200 };
+    },
+  );
+
+  app.post(
+    "/ekleYardimEtKaydi",
+    {
+      schema: {
+        body: {
+          type: "object",
+          properties: {
+            postId: { type: "string" },
+            adSoyad: { type: "string" },
+            sonDurum: { type: "string" },
+            email: { type: "string" },
+            aciklama: { type: "string" },
+            telefon: {
+              type: "string",
+              pattern: '^((?:\\+[0-9][-\\s]?)?\\(?0?[0-9]{3}\\)?[-\\s]?[0-9]{3}[-\\s]?[0-9]{2}[-\\s]?[0-9]{2})$'},
+          },
+          required: ["postId", "adSoyad", "sonDurum", "telefon", "aciklama"],
+        },
+      },
+    },
+    async (req, res) => {
+      await checkConnection();
+      await yardimEtPostIdExists(req, res);
+
+        const newYardimEtKaydi = new YardimEtKaydi({
+          postId: req.body.postId,
+          adSoyad: req.body.adSoyad,
+          telefon: req.body.telefon,
+          sonDurum: req.body.sonDurum,
+          email: req.body.email || "",
+          aciklama: req.body.aciklama,
+        });
+        await newYardimEtKaydi.save();
+
+
```

Comment:
linter ve formatter calistirdin mi?

Reply:
Hayır maalesef süreç değişikliğini şimdi okudum

Reply:
npm run lint && npm run format

> deprem-io-backend@1.0.0 lint
> npx rome check --apply-suggested .

Fixed 22 file(s) in 91ms

> deprem-io-backend@1.0.0 format
> npx rome format --write .

Formatted 22 file(s) in 51ms 👍